components/brave_rewards/browser/database/database_unblinded_token.cc

```diff
@@ -170,10 +170,10 @@ bool DatabaseUnblindedToken::MigrateToV15(sql::Database* db) {
   return true;
 }
 
-bool DatabaseUnblindedToken::InsertOrUpdate(
+bool DatabaseUnblindedToken::InsertOrUpdateList(
     sql::Database* db,
-    ledger::UnblindedTokenPtr info) {
-  if (!info) {
+    ledger::UnblindedTokenList list) {
```

Comment:
Should pass this as a const reference, since it's a container. This method and a few others in here.

Reply:
done

Reply:
I can't do cons ref afaik as this is vector of pointers and if I don't move list when passing around there will be pointer problem

Reply:
OK gotcha. Would a non-const reference work? It's not the end of the world if you can't, but it's usually bad form to pass a container by value, as it potentially entails a lot of copying.

Reply:
just ref should work yup

Reply:
but then I think we will have lint problems as it's enforced if you do ref it needs to be const

Reply:
OK, let's leave it as-is for now then since it's an urgent fix. Maybe something to revisit in the future.

Reply:
👍